keyboards/signum/3_0/teensy/teensy.h

```diff
@@ -16,4 +16,4 @@
 #pragma once
 
 #include "quantum.h"
```

Comment:
To be consistent within revisions, wouldnt either

1. `keyboards/signum/3_0/elitec/elitec.h` still includes `quantum.h`
2. `keyboards/signum/3_0/teensy/teensy.h` removes its include of `quantum.h`